cmd/draupnir-finalise-image

```diff
@@ -124,15 +124,7 @@ echo "Executing anonymisation script $ANON_FILE"
 sudo cat "$ANON_FILE" | sudo -u draupnir "$PSQL" -p "$PORT" --username=draupnir postgres
 
 echo "Vacuum all the databases in the cluster"
-sudo -u postgres bash <<BASH
-export PGPORT="$PORT"
-for database in \$(psql -t -c "select datname from pg_database where not datname like 'template%';");
-do
-  echo ">> vacuuming \$database"
-  psql --dbname "\$database" -t -c "select table_name from information_schema.tables where table_schema='public';" \
-    | xargs -P5 -n1 "$VACUUMDB" --dbname "\$database" --table
-done
-BASH
+sudo -u postgres vacuumdb --all --port="$PORT" --jobs="$(nproc)"
```

Comment:
Ah wait! `$VACUUMDB`

Reply:
Oops. Not really sure why we do this, as the `vacuumdb` command is provided on $PATH via a helper in Ubuntu (unlike `pg_ctl`), but updated for consistency anyway!

Reply:
I assume it's for multi-version installs, but given we don't parameterise it with a version it's not helping us so much.